### PR TITLE
Rethink vip app fields

### LINKED ISSUE #98: Rethink `vip app` and `vip app <app>` fields
What fields are important in the app list?

Apps:

* ID
* app name
* repo

Envs:

* environment name
* datacenter?
* branch? is branch different from environment name?
* current deploy?

### PR DESCRIPTION
**Pull request by joshbetz**
Uses better default fields as discussed in #98.

Fixes #98

### CODE REVIEW

File: src/bin/vip-app.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ command( { requiredArgs: 1, format: true } )
 		let res;
 
 		try {
-			res = await app( arg[ 0 ], 'id,name,environments{id,name,defaultDomain}' );
+			res = await app( arg[ 0 ], 'id,environments{name,datacenter,branch,currentCommit,defaultDomain}' );
```
Comment (mjangda) [inline src/bin/vip-app.js]: - `datacenter` isn't very useful.
- `branch` is a good idea but not as useful without the repo.
- `defaultDomain` should probably be renamed on display to primary domain?

Comment (mjangda) [reply src/bin/vip-app.js]: `currentCommit` is displaying as `[object Object]` for me.

**Review by mjangda — CHANGES_REQUESTED** [review]

File: src/bin/vip-app.js (hunk 1 above)
Comment (joshbetz) [reply src/bin/vip-app.js]: There’s a PR to fix the currentCommit https://github.com/Automattic/vip-go-api-public/pull/86

Good point about branch. I was also thinking about this in the context of app names — we don’t show the app name anywhere in vip app <app>. Maybe not quite as bad though; presumably you know what app you’ve specified?

**Comment by joshbetz** [comment]
As mentioned above, the current commit is actually broken in the API. There's a PR to get that fixed. Other than that, this is what it looks like now:

```
$ vip app vip-test
┌─────────┬───────────────────┬─────────┬─────────────────┬────────────────────────────┐
│ name    │ repo              │ branch  │ current commit  │ default domain             │
├─────────┼───────────────────┼─────────┼─────────────────┼────────────────────────────┤
│ testing │ wpcomvip/vip-test │ master  │ [object Object] │ vip-test.go-vip.co         │
├─────────┼───────────────────┼─────────┼─────────────────┼────────────────────────────┤
│ preprod │ wpcomvip/vip-test │ preprod │ [object Object] │ vip-test-preprod.go-vip.co │
└─────────┴───────────────────┴─────────┴─────────────────┴────────────────────────────┘
```

**Comment by joshbetz** [comment]
We have a dependency on https://github.com/Automattic/vip-go-api-public/pull/86 otherwise this is ready to go.

**Review by mjangda — APPROVED** [review]

**Pull request merged**